dms-core/src/main/kotlin/team/aliens/dms/domain/outing/service/GetOutingService.kt

```diff
@@ -18,4 +20,6 @@ interface GetOutingService {
     fun getAllOutingApplicationVOsBetweenStartAndEnd(start: LocalDate, end: LocalDate): List<OutingApplicationVO>
 
     fun getCurrentOutingApplication(studentId: UUID): CurrentOutingApplicationVO
+
+    fun getOutingHistoryByNameAndDateAndStatus(name: String?, date: LocalDate, outingStatus: OutingStatus): List<OutingHistoryVO>
```

Comment:
```suggestion
    fun getOutingHistoriesByStudentNameAndDateAndStatus(studentName: String?, date: LocalDate, outingStatus: OutingStatus): List<OutingHistoryVO>
```
histories, studentName이 좋을 것 같아요